packages/manager/modules/pci/src/projects/project/data-processing/submit-job/spark-config/spark-config.html

```diff
@@ -0,0 +1,120 @@
+        </label>
+    </oui-field>
+    <oui-field
+        ng-if="$ctrl.state.jobType === 'java' || $ctrl.state.jobType === 'python'"
+        data-label="{{ $ctrl.state.jobType === 'java'? 'data_processing_submit_job_stepper_spark_jar_file_label':'data_processing_submit_job_stepper_spark_python_file_label' | translate }}"
+        data-help-text="{{:: 'data_processing_submit_job_stepper_spark_jar_file_help' | translate }}" size="xl">
+        <input class="oui-input"
+               type="text"
+               id="main-application-code"
+               name="main-application-code"
+               data-ng-model="$ctrl.state.mainApplicationCode"
+               data-ng-change="$ctrl.onMainApplicationCodeChangeHandler()"
+               data-oui-autocomplete="$ctrl.containerObjects"
+               data-oui-autocomplete-property="name"
+               required
+               minlength="1"
+               maxlength="512">
+        <div class="oui-field__warning-show" data-ng-if="$ctrl.state.mainApplicationCodeFileNotFound">
+            <span ng-messages="mainApplicationCodeFileNotFound" role="alert">
+                <span class="oui-icon oui-icon-warning_circle" aria-hidden="true"></span>
+                <span>
+                    {{ ::'data_processing_submit_job_stepper_spark_not_found_file_label' | translate }}
+                </span>
+            </span>
+        </div>
+        <div class="oui-field__warning-show" data-ng-if="$ctrl.state.mainApplicationCodeFileInvalid">
+            <span ng-messages="mainApplicationCodeFileInvalid" role="alert">
+                <span class="oui-icon oui-icon-warning_circle" aria-hidden="true"></span>
+                <span>
+                    {{ ::'data_processing_submit_job_stepper_spark_invalid_jar_label' | translate }}
+                </span>
+            </span>
+        </div>
+    </oui-field>
+    <oui-field data-label="{{:: 'data_processing_submit_job_stepper_spark_main_class_label' | translate }}"
+               data-help-text="{{:: 'data_processing_submit_job_stepper_spark_main_class_help' | translate }}"
+               size="xl"
+               ng-if="$ctrl.state.jobType === 'java'">
+        <input class="oui-input"
+               type="text"
+               id="main-class"
+               name="main-class"
+               data-ng-model="$ctrl.state.mainClass"
+               data-ng-change="$ctrl.onMainClassChangeHandler()"
+               required
+               minlength="2"
+               maxlength="512">
+    </oui-field>
+    <span>
+     {{:: 'data_processing_submit_job_stepper_spark_arguments_label' | translate }}
+</span>
+    <oui-chips class="sparkconfig__arguments"
+               data-items="$ctrl.state.arguments"
+               data-on-remove="$ctrl.state.arguments=items"
+               data-stacked
+               data-closable>
+    </oui-chips>
+    <oui-field size="xl">
+        <input class="oui-input"
```

Comment:
Won't `oui-input` suit your need ?

Reply:
@FredericEspiau there is no `oui-input` component